packages/react/README.md

```diff
@@ -34,7 +35,7 @@ yarn add carbon-components-react carbon-components carbon-icons
    [`carbon-components-react/.storybook/polyfills.js` file](./.storybook/polyfills.js)
    is required.
 
-If you just want to try out `carbon-components-react`, you can also use
+If you want to try out `carbon-components-react`, you can also use
```

Comment:
Quick note:

Locally when I ran `yarn format` & then tried to commit the changes, I got the following pre-commit error:

```
packages/react/README.md
38:8-38:12  warning  `just` may be insensitive, try not to use it  just  retext-equality
```

So that's why I removed the word. 👍 Hopefully that's all right with you all?

Reply:
Sounds perfect, thanks!